### PR TITLE
chore: add metadata to csproj file for NuGet

### PR DESCRIPTION
**Pull request by pgautier404**
Add extended metadata for display on the public NuGet package repository.

### CODE REVIEW

File: .github/workflows/push.yml [hunk 1]
```diff
@@ -188,10 +174,14 @@ jobs:
             VERSION="${{needs.release.outputs.version}}"
             echo "version: ${VERSION}"
             dotnet build --configuration Release
-            nuget pack -Version ${VERSION} -Properties Configuration=Release
-            nuget push Momento.${VERSION}.nupkg -Source Artifactory -Verbosity detailed
+            dotnet pack -property:Version=${VERSION} --configuration Release
+            pwd
```
Comment (malandis) [inline .github/workflows/push.yml:178]: Is this line just for debug output?

File: csharp/Momento.csproj [hunk 2]
```diff
@@ -2,6 +2,16 @@
...
+    <PackageId>Momento.Protos</PackageId>
+    <Authors>Momento</Authors>
+    <Company>Momento Inc</Company>
+    <Description>
+      Generated gRPC and Protocol Buffers types for Momento clients
+    </Description>
+    <PackageTags>caching, cache, serverless, key value, simple caching service</PackageTags>
+    <Copyright>Copyright (c) Momento Inc 2022</Copyright>
+    <PackageLicenseExpression>Apache-2.0</PackageLicenseExpression>
+    <ProjectUrl>https://github.com/momentohq/client_protos/tree/main/csharp</ProjectUrl>
```
Comment (malandis) [inline csharp/Momento.csproj:14]: The URL should just be to /client_protos/

**Review by malandis — CHANGES_REQUESTED** [review]

File: .github/workflows/push.yml (hunk 1 above)
Comment (pgautier404) [reply .github/workflows/push.yml:178]: Ah, yes, will remove. Thanks.

File: csharp/Momento.csproj (hunk 2 above)
Comment (pgautier404) [reply csharp/Momento.csproj]: I modeled that after the pypi info, but I tend to agree with you. I've changed it.

**Review by malandis — APPROVED** [review]

**Pull request merged**